osu.Game.Tests/osu.Game.Tests.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\osu.TestProject.props" />
   <ItemGroup Label="Package References">
+    <PackageReference Include="Bogus" Version="35.6.2" />
```

Comment:
Probably gonna need @peppy's ok for the inclusion on this. It seems mostly legit on a quick look although there's some [DRM-shaped stuff happening](https://github.com/bchavez/Bogus/blob/master/Source/Bogus/Premium/LicenseVerifier.cs#L98) because the library has paid extensions included, and also some other [somewhat](https://github.com/bchavez/Bogus/blob/a82ea298c20e0a6be08c6b5861546fb6701f424e/Source/Bogus/DataSets/Internet.cs#L31-L39) [dodgy](https://github.com/bchavez/Bogus/issues/343) stuff.

Reply:
One alternative that just came to my mind, is I can replace it with

```
public MultiplayerPlaylistItem Clone()
{
    MultiplayerPlaylistItem clone = (MultiplayerPlaylistItem)MemberwiseClone();
    clone.RequiredMods = RequiredMods.ToArray();
    clone.AllowedMods = AllowedMods.ToArray();
    return clone;
}
```

And we can live without tests under the premise that `MemberwiseClone()` has been properly tested by .NET itself. Thoughts?

Reply:
Ah but the tests also test MPI -> PI -> MPI... Think I'll replace with MemberwiseClone() in any case. :/

Reply:
I think it's okay since it's limited to the tests project.